config/manager/patches/manager_metrics_port_patch.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        ports:
+          - containerPort: 8443
+            name: https
+            protocol: TCP
```

Comment:
File is missing a trailing newline character

Reply:
Thanks. Fixed.